doc/developer_installation_on_ubuntu.md

```diff
@@ -92,8 +86,16 @@ on the system, so we will have to install one:
 
 - `sudo apt-get install libqtwebkit-dev`
 
+Clone the ontohub sources:
+- `git clone git@github.com:ontohub/ontohub.git`
+Note that you need to get access rights from the Ontohub team first.
```

Comment:
Actually a clone via ssh/git still works if you do not have write-permissions on a repository. Only the push will later fail (as it should).